src/Datagrid/ProxyQuery.php

```diff
@@ -134,4 +139,20 @@ public function getMaxResults(): ?int
     {
         return $this->maxResults;
     }
+
+    /**
+     * @return mixed[]
+     */
+    public function getOptions(): array
+    {
+        return $this->options;
+    }
```

Comment:
Looks like we don't need this method to be called outside the class, we can call `$this->options` instead.